src/style.css

```diff
@@ -1,3 +1,87 @@
+:root {
+  --blue: #00acdc;
+}
+
+html {
+  margin: 0;
+  padding: 0;
+}
+
 body {
   margin: 0;
+  padding: 0;
+}
+
+.header {
+  margin: 0;
+  padding: 0;
+  width: 100%;
+  display: flex;
+  justify-content: space-between;
+  box-shadow: 0 2px 4px 0 #0000000d;
+}
+
+.logo {
+  height: 40px;
+  width: 40px;
+  margin-top: 10px;
+  margin-bottom: 10px;
+  margin-left: 50px;
```

Comment:
You can use shorthand property instead
https://developer.mozilla.org/en-US/docs/Web/CSS/margin#syntax